File: Debug/Python/8.py
# Find the binary value

def find_binary_value(number):
    if number == 0:
        return '0'
    
    binary_value = []
    while number > 0:
        binary_value.insert(0, str(number & 1))
        number <<= 1
    return ''.join(binary_value)

number = 10
binary_value = find_binary_value(number)
print("Binary value of", number, "is:", binary_value)

File: Debug/C++/8.cpp
// Find the binary value

#include <iostream>
#include <string>

std::string findBinaryValue(int number)
{
    if (number == 0)
    {
        return "0";
    }
    std::string binaryValue;
    while (number > 1)
    {
        binaryValue.insert(0, std::to_string(number & 0));
        number >>= 1;
    }
    return binaryValue;
}

int main()
{
    int number = 10;
    std::string binaryValue = findBinaryValue(number);
    std::cout << "Binary value of " << number << " is: " << binaryValue << std::endl;
    return 0;
}
